<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="username_edit"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="connect_button">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="username_label">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="port_label">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="address_label">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="address_edit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="port_edit"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="msg_list"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="msg_edit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="send_button">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
